### ISSUE TITLE
Has anyone successfully run the node on a Windows Server 2019 system? Please share your tutorial. thank you very much.

### ISSUE THREAD

**Issue by Wu170** [pi-node/instructions#29]

**Comment by ihatejam** [comment]
It should work, but the node component of the Pi Node Software runs on Docker. While Docker is free to run on a Windows desktop OS, I'm not sure if there is a free option for running Docker on the Windows Server platform.
https://github.com/docker/labs/blob/master/windows/windows-containers/README.md 

It's probably best to run a Windows desktop OS and use install either Docker Desktop (preferred) or Docker Toolbox in order to run the Node containers.

**Comment by Wu170** [comment]
Yes, Windows Server 2019 system should be able to run nodes. We should test the running nodes of Windows Server 2019. Because the Windows Server 2019 system belongs to a kind of server system, if it can run the node, it should be better than that of win10 system. This is the configuration website of docker running on Windows Server 2019 system; https://hub.docker.com/editions/enterprise/docker-ee-server-windows
